README.md

```diff
@@ -5,4 +5,4 @@
 2. Arithmetic operations
 3. Execution
 4. Error handling
-5. Unit tests
\ No newline at end of file
+5. Unit tests
```

Comment:
I noticed your current branch name is main. I’d recommend using separate branch names for each step, such as step1, step2, etc.